pl/code_editor/README.md

```diff
@@ -2,7 +2,7 @@
 
 Zaraz napiszesz swoją pierwszą linijkę kodu, czas na zainstalowanie edytora tekstu!
 
-Tylko który wybrać? To nie takie proste, bo wybór zależy w dużej mierze od tego, co oczekujesz od swojego edytora. Większość programistów Pythona używa bardzo skomplikowanych i niezwykle bogatych w różne funkcjonalności IDE (z ang. Integrated Development Environments - zintegrowane środowisko deweloperskie), takich jak: PyCharm. Jako osoba początkująca nie potrzebujesz jednak aż tak skomplikowanego edytora. Wystarczy coś prostszego, ale nadal dostatecznie użytecznego.
+Tylko który wybrać? To nie takie proste, bo wybór zależy w dużej mierze od tego, co oczekujesz od swojego edytora. Większość programistów Pythona używa bardzo skomplikowanych i niezwykle bogatych w różne funkcjonalności IDE (z ang. Integrated Development Environments - zintegrowanych środowisk deweloperskich), takich jak np. PyCharm. Jako osoba początkująca nie potrzebujesz jednak aż tak skomplikowanego edytora. Wystarczy coś prostszego, ale nadal dostatecznie użytecznego.
```

Comment:
I will go with Polish here, as this is Polish terms :)

Zmieniłabym tutaj kolejność danych w nawiasie, w zdaniu "Większość programistów Pythona używa...", bo teraz to brzmi trochę dziwnie ;)

Napisałabym:

> Większość programistów Pythona używa bardzo skomplikowanych i niezwykle bogatych w różne funkcjonalności IDE (z ang. zintegrowanych środowisk programistycznych - Integrated Development Environments), takich jak np. PyCharm.

I will merge your PR and change just this sentence :)